src/BookingRow.js

```diff
@@ -1,13 +1,27 @@
-import React from "react";
+import React, { useState } from "react";
 import moment from "moment";
 
 function calcDeff(chkIn, chkOut) {
   return moment(chkOut, "YYYY-MM-DD").diff(moment(chkIn, "YYYY-MM-DD"), "days");
 }
 
 const BookingRow = props => {
+  const [bookingRowClass, setBookingRowClass] = useState("");
+  const [selected, setSelected] = useState(false);
+
+  function selectRow() {
+    // console.log("someone calls me");
+    if (selected === false) {
+      setSelected(true);
+      setBookingRowClass("search-selected");
```

Comment:
This is correct, but some good React advice is to use as little state as possible. So here, we can calculate what the `bookingRowClass` should be based on the `selected` state. This means that if we ever want to change the class to something different, we only need to change it in 1 place. Does that make sense?